integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -112,7 +111,7 @@ private void exportUserJourney(
 
     // Export workspace dataset to workspace
     var workspaceId = UUID.fromString(workspaceDest.getWorkspaceId());
-    datasetsApi.exportDataset(datasetId, workspaceId);
+    datasetsApi.exportDataset(datasetId, new DatasetExportRequest().workspaceId(workspaceId));
```

Comment:
My usual preference is to avoid local variables unless I think they make the code clearer. In general I'm not a fan of code that only does one thing per line and creates a new variable for each indeterminate step. On the other hand, writing code as a single line of text can obscure meaning and make the code hard to follow. So it's hard to say exactly when I think one is better or the other is, and the answer is probably different for everyone...

In this case I don't think it's necessary to introduce another variable. it would be great if we had better constructors for objects like `DataExportRequest`; I think the fluent style doesn't add much to code like this. But this is a swagger generated object so we're stuck with this API (or we need to do a lot of templating work to fix it).